src/backend/core/api/serializers.py

```diff
@@ -26,6 +26,32 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+class IdentitySerializer(serializers.ModelSerializer):
+    """Serializes user identities"""
+
+    class Meta:
+        model = models.Identity
+        fields = ["email", "first_name", "last_name"]
+        read_only_fields = ["email", "first_name", "last_name"]
+
+
+class UserListSerializer(serializers.ModelSerializer):
+    """Serialize users for list views, including nested serializers."""
+
+    main_identity = IdentitySerializer(read_only=True)
```

Comment:
I don't think you should expose the identity object to the API. 
The first_name of the main identity is the user's first name and is readonly so I would say we want to return:
```
{
    "id": "xyz",
    "user": {
        "id": "xyz",
        "email": "david.bowman@example.com",
        "first_name": "david",
        "last_name": "bowman",
    },
    "role": "member",
    "abilities": {...},
}
```
The "email" field will always come from the identity as users with an email set on the User model are filtered out of the API.

Reply:
I agree, that's what I intended to do. I'm just making my way step by step.